main.js

```diff
@@ -0,0 +1,102 @@
+let companies = [
+  { img: "http://placehold.it/140x100", name: "Walmart", address: "10000 Harbor Santa Ana, CA", review: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure" },
+  { img: "http://placehold.it/140x100", name: "Target", address: "1000 Seal Beach Blvd. Seal Beach, CA", review: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure" },
+  { img: "http://placehold.it/140x100", name: "7-11", address: "100 Barranca Irvine, CA", review: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure" }
+];
+
+// The DOM_ sign preceding a variable name is a naming convention that represents a DOM element.
```

Comment:
This is a good comment! You are explaining the significance of a convention that the reader might now be aware of.